.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,108 @@
+name: Bug report
+description: Create a report to help us improve
+title: "BUG - <title>"
+labels: ["type:Bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi! Thanks for using the Jupyter Docker Stacks and taking some time to contribute to this project.
+
+        We'd appreciate it if you could check out the [Troubleshooting common problems](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/troubleshooting.html) section in the documentation,
+        as well as [existing issues](https://github.com/jupyter/docker-stacks/issues) prior to submitting an issue to avoid duplication.
+
+        Please answer the following sections to help us troubleshoot the problem.
+
+  - type: dropdown
+    id: docker_images
+    attributes:
+      label: What docker image are you using?
+      description: Select as many images as applicable
+      multiple: true
+      options:
+        - all-spark-notebook
+        - base-notebook
+        - datascience-notebook
+        - minimal-notebook
+        - pyspark-notebook
+        - r-notebook
+        - scipy-notebook
+        - tensorflow-notebook
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: OS system you are using
+      placeholder: |
+        Example:
+        Linux - Ubuntu
```

Let's be more precise here - there are actually 2 operating systems involved.
One - running the docker and jupyter, and the second one - running the web browser.
I think it would be better to have something like this:

```suggestion
      label: OS system and architecture running docker image
      placeholder: |
        Example:
        Ubuntu/aarch64
```